nlgeval/tests/test_nlgeval.py

```diff
@@ -103,6 +103,36 @@ def test_compute_metrics_omit(self):
         self.assertAlmostEqual(0.960771, scores['GreedyMatchingScore'], places=5)
         self.assertEqual(7, len(scores))
 
+    def test_compute_metricsa(self):
+        n = NLGEval()
+
+        # # Individual Metrics
+        scores = n.compute_individual_metrics(ref=["Chocolate , a New Way to Get Cancer "],
+                                              hyp="Chocolate , a Healthy Food")
+        print("Chocolate , a New Way to Get Cancer ", scores)
+
+        scores = n.compute_individual_metrics(ref=["Chocolate and an Interview "],
+                                              hyp="Chocolate , a Healthy Food")
+        print(scores)
```

Comment:
Can you assert something here, so that tests can detect regressions?

Reply:
I found that this case is not necessary. I will remove it